Source/JavaScriptCore/runtime/ArrayBuffer.h

```diff
@@ -268,7 +273,8 @@ class ArrayBuffer final : public GCIncomingRefCounted<ArrayBuffer> {
     JS_EXPORT_PRIVATE static Ref<ArrayBuffer> create(std::span<const uint8_t> = { });
     JS_EXPORT_PRIVATE static Ref<ArrayBuffer> create(ArrayBufferContents&&);
     JS_EXPORT_PRIVATE static Ref<ArrayBuffer> createAdopted(std::span<const uint8_t>);
-    JS_EXPORT_PRIVATE static Ref<ArrayBuffer> createFromBytes(const void* data, size_t byteLength, ArrayBufferDestructorFunction&&);
+    JS_EXPORT_PRIVATE static Ref<ArrayBuffer> createFromBytes(std::span<const uint8_t> data, ArrayBufferDestructorFunction&&);
+    static Ref<ArrayBuffer> createFromBytes(const void* data, size_t byteLength, ArrayBufferDestructorFunction&&);
```

Comment:
Are there a lot of clients of this that can’t be quickly converted?

Reply:
There are not many, just 3 iirc. However, all of them have a void* and a length. As a result, I would have to construct a span and to a static cast. It will make the call sites look a little worse but it would allow us to drop the overload. I hesitated.

Reply:
I ported the 3 remaining call sites.